packages/melody-streams/src/component.js

```diff
@@ -61,7 +62,7 @@ Object.assign(Component.prototype, {
                         ...options,
                         detail,
                     });
-                    this.el.dispatchEvent(event);
+                    _this.el.dispatchEvent(event);
```

Comment:
Mhh... what about changing the implementation of `dispatchCustomEvent` to an arrow function instead?

Reply:
I accidentally fixed the same issue in another PR #117 with an arrow function.